Lib/test/test_string_literals.py

```diff
@@ -177,7 +177,7 @@ def test_eval_str_invalid_octal_escape(self):
     def test_invalid_escape_locations_with_offset(self):
         with warnings.catch_warnings(record=True) as w:
             warnings.simplefilter('always', category=SyntaxWarning)
-            eval("\"'''''''''''''''''''''invalid\ Escape\"")
+            eval("\"'''''''''''''''''''''invalid\\ Escape\"")
```

Comment:
Isn't the purpose of the test to emit a SyntaxWarning warning?

Reply:
Oh, that's confusing :-) But you're right, the code emits the warning when the code is compiled, not when eval() is called.